ompi/mca/coll/basic/coll_basic_gatherv.c

```diff
@@ -44,7 +44,7 @@ mca_coll_basic_gatherv_intra(const void *sbuf, int scount,
                              struct ompi_communicator_t *comm,
                              mca_coll_base_module_t *module)
 {
-    int err, i, peer, rank, size;
+    int err, peer, rank, size;
```

Comment:
Can you keep `int i` declaration on the top, and change `size_t i` to `size_t j`?